bookwyrm/templates/preferences/edit_user.html

```diff
@@ -34,29 +34,29 @@ <h2 class="title is-4">{% trans "Profile" %}</h2>
                 <div class="column">
                     {{ form.avatar }}
                     {% for error in form.avatar.errors %}
-                    <p class="help is-danger">{{ error | escape }}</p>
+                    <p class="help is-danger" id="desc_avatar">{{ error | escape }}</p>
                     {% endfor %}
                 </div>
             </div>
             <div class="field">
                 <label class="label" for="id_name">{% trans "Display name:" %}</label>
                 {{ form.name }}
                 {% for error in form.name.errors %}
-                <p class="help is-danger">{{ error | escape }}</p>
+                <p class="help is-danger" id="desc_name">{{ error | escape }}</p>
                 {% endfor %}
             </div>
             <div class="field">
                 <label class="label" for="id_summary">{% trans "Summary:" %}</label>
                 {{ form.summary }}
                 {% for error in form.summary.errors %}
-                <p class="help is-danger">{{ error | escape }}</p>
+                <p class="help is-danger" id="desc_summary">{{ error | escape }}</p>
                 {% endfor %}
             </div>
             <div class="field">
                 <label class="label" for="id_email">{% trans "Email address:" %}</label>
                 {{ form.email }}
                 {% for error in form.email.errors %}
-                <p class="help is-danger">{{ error | escape }}</p>
+                <p class="help is-danger" id="desc_email">{{ error | escape }}</p>
```

Comment:
I think these ids should be in a parent element, so that when there are multiple errors, they don't have duplicate ids

Reply:
I hadn't thought about that, thank you!